groupease-server/src/main/java/io/github/groupease/model/ChannelInvitation.java

```diff
@@ -1,8 +1,10 @@
 package io.github.groupease.model;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import org.hibernate.annotations.UpdateTimestamp;
 import javax.persistence.*;
 import java.time.Instant;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Is this something we need to implement across the board?

Reply:
Yes, on any object that can be received from the client (or nested within an object from the client).